cli/pom.xml

```diff
@@ -137,8 +137,12 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <!-- Sets the VM argument line used when unit tests are run. -->
+                    <!-- XPath expression and operations limits (jdk.xml.*) are disabled as the application
+                    does not support processing externally/user-provided XPath expressions.-->
                     <argLine>-Dspring.config.additional-location=file://${user.home}/.l10n/config/cli/
                         -Dspring.profiles.active=${user.name},test -Xmx1024m
+                        -Djdk.xml.xpathExprGrpLimit=0 -Djdk.xml.xpathTotalOpLimit=0 -Djdk.xml.xpathExprOpLimit=0
```

Comment:
Yes, as the tests don't use the Application entry points / main().

Reply:
Maybe this should should be move to a static initializer in the AndroidFilter, if that's the class that is the source of causing the problem. This way there will be only one place where it is set and it will be related to the source of the issue.

Reply:
It's not only AndroidFilter that suffers from this problem, we still need to cover the scenarios where this exception happens without the use of the AndroidFilter at all.